airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/spec.json

```diff
@@ -36,6 +36,18 @@
                 "const": "oauth2.0",
                 "order": 0
               },
+              "client_id": {
+                "type": "string",
+                "title": "Client ID",
+                "description": "The client ID of the Zendesk Support OAuth Client.",
+                "airbyte_secret": true
+              },
+              "client_secret": {
+                "type": "string",
+                "title": "Client secret",
+                "description": "The client secret of the Zendesk Support OAuth Client.",
+                "airbyte_secret": true
+              },
```

Comment:
I'm confused, what does adding oauth support have to do with updating the cursor for the tickets stream?